spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -489,6 +520,51 @@ class Analyzer(tableUtils: TableUtils,
     }
   }
 
+  // For groupBys validate if the timestamp provided produces some values
+  // if all values are null this should be flagged as an error
+  def runTimestampChecks(df: DataFrame, sampleFraction: Double = 0.1): Map[String, String] = {
```

Comment:
0.1 seems a large number. Let's change it to a smaller number with a upper bound if possible.

Reply:
maybe we could do sample or first 1000 rows

Reply:
done.